CHANGELOG.md

```diff
@@ -4,7 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.5.1] - Unreleased
+## [0.5.1] - 2023-05-05
+### Added
+- Add global event listener
```

Comment:
Add a new line between different section
```suggestion
- Add global event listener

```

Reply:
Added a new line and changed the message.